PythonTool/TagExpansion.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import numpy as np
+import os
+from pymoab import core, tag, types
+
+
+def parse_arguments():
+    """
+    Parse the argument list and return a mesh file location.
+
+    Input:
+    ______
+       none
+
+    Returns:
+    ________
+       args: Namespace
+           User supplied mesh file location.
+    """
+
+    parser = argparse.ArgumentParser(description="Expand vector tags to scalar tags.")
+
+    parser.add_argument("meshfile",
+                        type=str,
+                        help="Provide a path to the mesh file."
+                        )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def tag_expansion(mesh_file):
```

Comment:
I agree that this is a good breakdown when handling multiple vector tags of different lengths! This will come in a future PR. For now, I created a separate function to return the list of tags on a specific element in the mesh!